core/dbt/cli/main.py

```diff
@@ -608,7 +608,7 @@ def retry(ctx, **kwargs):
     task = RetryTask(
```

Comment:
Let's also remove @requires.manifest & manifest from the `RetryTask` constructor